pandas/_libs/groupby.pyx

```diff
@@ -1270,11 +1274,14 @@ cdef group_cummin_max(groupby_t[:, ::1] out,
     cdef:
         Py_ssize_t i, j, N, K, size
         groupby_t val, mval
-        ndarray[groupby_t, ndim=2] accum
+        groupby_t[:, ::1] accum
         intp_t lab
+        bint val_is_nan, use_mask
+
+    use_mask = mask is not None
 
     N, K = (<object>values).shape
-    accum = np.empty((ngroups, K), dtype=np.asarray(values).dtype)
+    accum = np.empty((ngroups, K), dtype=np.asarray(values).dtype, order='C')
```

Comment:
Does this matter? (I think that `order="C"` is the default?)

Reply:
This was just an attempt to be super explicit because of specifying contiguity for `accum`.  Can remove though since is default like you mention

Reply:
> super explicit

i like this instinct.  im mostly indifferent as to whether to include the "order='C'" here, lean against bc in 6 months I'm going to read this file and ask "why are we specifying it here and not there" and then have to go look up the default